### PR TITLE
Port store fsm step 2

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: tikv/raftstore/msg.go [hunk 1]
```diff
@@ -90,9 +90,8 @@ const (
 	StoreTickCompactCheck     StoreTick = 0
 	StoreTickPdStoreHeartbeat StoreTick = 1
 	StoreTickSnapGC           StoreTick = 2
-	StoreTickCompactLockCF    StoreTick = 3
-	StoreTickConsistencyCheck StoreTick = 4
-	StoreTickCleanupImportSSI StoreTick = 5
+	StoreTickConsistencyCheck StoreTick = 3
+	StoreTickCleanupImportSSI StoreTick = 4
```
Comment (ngaut) [inline tikv/raftstore/msg.go]: SST?

File: tikv/raftstore/fsm_store.go [hunk 2]
```diff
@@ -157,69 +178,193 @@ func (s *storeFsm) isStopped() bool {
...
-func (p *raftPoller) handleRaftReady() {
-	return // TODO: stub
+func (p *raftPoller) handleRaftReady(peers []fsm) {
+	if len(p.pendingProposals) > 0 {
+		for _, proposal := range p.pendingProposals {
+			msg := Msg{Type: MsgTypeApplyProposal, Data: proposal}
+			p.pollCtx.applyRouter.scheduleTask(proposal.RegionId, msg)
+		}
+	}
+	kvWB := p.pollCtx.kvWB
```
Comment (zhangjinpeng1987) [inline tikv/raftstore/fsm_store.go]: Has abandoned `trans->flush` here?

Comment (coocood) [reply tikv/raftstore/fsm_store.go]: yes

File: tikv/raftstore/fsm_store.go [hunk 3]
```diff
@@ -253,36 +397,176 @@ type senderPeerFsmPair struct {
...
+		log.Infof("region %d is applying snapshot", region.Id)
+		sender, peer, err := createPeerFsm(storeID, b.cfg, b.regionScheduler, b.engines, region)
+		if err != nil {
+			return nil, err
+		}
+		peer.scheduleApplyingSnapshot()
+		meta.regionRanges.Insert(region.EndKey, regionIDToBytes(region.Id))
+		meta.regions[region.Id] = CloneRegion(region)
+		regionPeers = append(regionPeers, senderPeerFsmPair{sender: sender, fsm: peer})
+	}
```
Comment (zhangjinpeng1987) [inline tikv/raftstore/fsm_store.go]: Forget call `clearStaleData` here.

Comment (coocood) [reply tikv/raftstore/fsm_store.go]: badger doesn't support this feature.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**